.github/workflows/dockerhub-description.yml

```diff
@@ -0,0 +1,962 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Update Docker Hub Description
+on:
+  schedule:
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+jobs:
+  dockerHubDescription:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Description for audioqna
+      uses: peter-evans/dockerhub-description@v4
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+        repository: opea/audioqna
+        short-description: "The docker image served as an audioqna gateway and using language modeling to generate answers to user queries by converting audio input to text, and then using text-to-speech (TTS) to convert those answers back to speech for interaction."
```

Comment:
These descriptions are for DockerHub, they do not need to mention they are docker images, that's already known.

And past tense (`served`, `acted`, `powered`, `exposed`) sounds really odd in these description.  Don't those images work anymore?

 Something like this would be more to the point for all of these descriptions:
> AudioQnA gateway. Uses language modeling to interact with the user, first by converting audio input queries to text, and then using text-to-speech (TTS) to convert those answers back to speech.